src/entities/Task/model/types/task.ts

```diff
@@ -0,0 +1,26 @@
+export enum StatusEnum {
+    COMPLETED = 'COMPLETED',
+    TO_DO = 'TO_DO',
+    IN_PROGRESS = 'IN_PROGRESS'
+}
+
+export interface SubTask {
+    id: string;
+    taskId: string;
+    title: string;
+    completedAt: number | null;
+    status: StatusEnum;
+}
+
+export interface Task {
```

Comment:
`Task` and `SubTask` interfaces have common properties which can be described in a separate interface and then you need just to extend from it